OamLog.yaml

```diff
@@ -154,35 +154,55 @@ paths:
                       type: object
```

Comment:
We have a placeholder for client-application's release-number in client UUID's . In order to make it more generic, can we make the client-application's release-number to --* instead of 2-0-0 or 1-0-0.
Note : similar change made in https://github.com/openBackhaul/RegistryOffice/pull/306#discussion_r1050414200

Reply:
If this change would not make any difference in further processing of the specification, I would like to cut investing time into cosmetic changes of UUIDs.